### ISSUE TITLE
Sort order

### ISSUE THREAD

**Issue by agzam** [m2ym/direx-el#55]
For some reason directory content on my machine is always wacky.
![image](https://user-images.githubusercontent.com/1091022/31595755-bce18ff8-b1f2-11e7-907d-a2fc0de1a22d.png)

How can I fix this?

**Comment by sadhu89** [comment]
+1 for this. Did you find a solution @agzam?

**Comment by agzam** [comment]
@sadhu89 no, unfortunately I have not found a way to solve this. But I have discovered `direx-grep` package that makes it easy to search for things. One another thing would be awesome to have - ability to treat any node as "root", hiding other less relevant at the moment
